.travis/linux_openresty_runner.sh

```diff
@@ -108,6 +108,9 @@ script() {
     openresty -V
     sudo service etcd start
 
+    # set password for redis-server
+    echo "CONFIG SET requirepass 'foobared'" | redis-cli
```

Comment:
IMO, you should add Redis password in the related test cases, not by default.
We can use `lua-resty-redis` to call `set_redis_password` Lua method in the separate test case file, and clear in the last test case.

Reply:
> set_redis_password

I can not find method like `set_redis_password ` in `lua-resty-redis` repo. ;-(

Reply:
> > set_redis_password
> 
> I can not find method like `set_redis_password ` in `lua-resty-redis` repo. ;-(

my fault, the name is just a example.